src/cljs/rems/profile.cljs

```diff
@@ -95,12 +121,47 @@
                   [:button.btn.btn-primary
                    {:type "submit"}
                    (text :t.profile/save)]])}]
+
      [:div.mt-3
       [collapsible/component
        {:title (text :t.profile/your-details)
         :always [:<>
                  [user/username (:user identity)]
-                 [user/attributes attributes false]]}]]]))
+                 [user/attributes (:user identity) false]]}]]
+
+     (when (and (:enable-ega config) (roles/has-roles? :handler))
+       [:div.mt-3
+        [collapsible/component
+         {:title (text :t.profile/ega-full)
+          :always (if @(rf/subscribe [::user-settings :fetching?])
+                    [spinner/big]
+                    [:<>
+                     (text :t.profile/ega-intro)
+                     (if-let [ega-api-key-expiration-date (get-in @(rf/subscribe [::user-settings])
+                                                                  [:ega :api-key-expiration-date])]
```

Comment:
big then and else cases in this if, might be easier to read if they were given names. I mean something like:
```
(if-let [expiration-date ...]
  [update-or-delete-ega-key]
  [generate-ega-key])
```

Reply:
Ok, tried something.